src/components/Home.vue

```diff
@@ -1,14 +1,11 @@
 <script setup>
 import { defineProps } from "vue";
 import { ref } from "vue";
-
 const props = defineProps({
   tasks: Array,
 });
-
 let currentTaskIndex = 0;
```

Comment:
I'm thinking we should the ref for this currentTaskIndex and use the watchEffect to update the currentTask, here's what I mean:

```
const currentTaskIndex = ref(0);
const currentTask = ref(null);
watchEffect(() => {
  currentTask.value = props.tasks[currentTaskIndex.value];
})
function getNextTask() {
  currentTaskIndex.value++;
  if (props.tasks.length <= currentTaskIndex) {
    // for now it will start at 0 but when connected to backend we can fetch a batch of tasks
    currentTaskIndex = 0;
  }
}
```

what do you think?

Reply:
This one is good it works nicely

Reply:
great, please not that the currentTaskIndex update in the conditional should be: `currentTaskIndex.value = 0`

Reply:
oh yeah I noticed it was missing the .value

Reply:
thank you!